### PR TITLE
[docs] Improve bash-completion instructions

### PR DESCRIPTION
**Pull request by ultimike**
## The Issue
Snippet to add to .bash_profile was incorrect.

## How This PR Solves The Issue
Corrected the incorrect snippet.

## Release/Deployment Notes

<a href="https://gitpod.io/#https://github.com/drud/ddev/pull/4638"></a>

### CODE REVIEW

File: docs/content/users/install/shell-completion.md [hunk 1]
```diff
@@ -8,14 +8,14 @@ Shells like bash and zsh need help to do this though, they have to know what the
...
-    ```bash
-    [[ -r "/usr/local/etc/profile.d/bash_completion.sh" ]] && . "/usr/local/etc/profile.d/bash_completion.sh"
-    ```
-
+    The easiest way to use bash completion on macOS is install it with Homebrew. `brew install bash-completion`. When you install it though, it will warn you with something like this, which **may vary on your system**.
+    
+    > Add the following line to your ~/.bash_profile:
+    > 
+    >  ```
+    >  [[ -r "/opt/homebrew/etc/profile.d/bash_completion.sh" ]] && . "/opt/homebrew/etc/profile.d/bash_completion.sh"
```
Comment (rfay) [inline docs/content/users/install/shell-completion.md:16]:
```suggestion
    >  [[ -r "$(brew --prefix)/etc/profile.d/bash_completion.sh" ]] && . "$(brew --prefix)/etc/profile.d/bash_completion.sh"
```

This is what it actually says and what belongs there.

File: docs/content/users/install/shell-completion.md [hunk 2]
```diff
@@ -8,14 +8,14 @@ Shells like bash and zsh need help to do this though, they have to know what the
...
+    >  ```
```
Comment (rfay) [inline docs/content/users/install/shell-completion.md:17]:
```suggestion
    ```
```

**Review by rfay — CHANGES_REQUESTED** [review]

File: docs/content/users/install/shell-completion.md (hunk 1 above)
Comment (ultimike) [reply docs/content/users/install/shell-completion.md:16]: Roger that.

Comment (rfay) [reply docs/content/users/install/shell-completion.md:16]: You can just commit it with the button there.

**Review by rfay — APPROVED** [review]
Thanks!

**Pull request merged**